blog/views.py

```diff
@@ -0,0 +1,58 @@
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import redirect
+from django.urls import reverse
+from django.views import generic
+from django.views.generic.edit import FormMixin
+from .forms import CommentaryForm
+
+from blog.models import Post
+
+
+class IndexView(generic.ListView):
+    model = Post
+    posts = Post.objects.all().order_by("-created_time")
```

Comment:
The 'posts' attribute is not used in the IndexView class. If you intended to use it, consider using it in the context or removing it if it's unnecessary.